db/seeds.rb

```diff
@@ -8,3 +8,4 @@
 require File.dirname(__FILE__) + '/seed_helper.rb'
 
 create_roles_and_permissions
+populate_languages
```

Comment:
I'm not sure - you'll need to work out what is called for each deploy and integrate it into there...